src/QsCompiler/LanguageServer/LanguageServer.cs

```diff
@@ -126,6 +126,25 @@ public void Dispose()
         internal Task PublishDiagnosticsAsync(PublishDiagnosticParams diagnostics) =>
             this.NotifyClientAsync(Methods.TextDocumentPublishDiagnosticsName, diagnostics);
 
+        internal void CheckDotNetSDKVersion()
+        {
+            var sdkVersions = DotNetSDKHelper.GetSDKVersions();
+            if (sdkVersions == null)
+            {
+                this.LogToWindow($"Unable to detect .NET SDK versions", MessageType.Error);
+            }
+            else
+            {
+                if (!sdkVersions.HasDotNet31)
+                {
+                    this.LogToWindow($".NET SDK 3.1 not found. QDK requires .NET SDK 3.1 to work properly.", MessageType.Error);
+                    this.ShowInWindow(
+                        $"QDK requires .NET SDK 3.1 to work properly. Please install .NET SDK 3.1 from https://dotnet.microsoft.com/download/dotnet-core/3.1",
```

Comment:
I think the rest of the QDK actually works when only .NET 5 is installed, it's only the language server that's broken.

Reply:
updated to "Quantum Development Kit Extension requires .NET Core SDK 3.1 to work properly".
- Spelled out QDK since users might not immediately recognize QDK.
- Added Extension, to clarify that the rest of QDK works fine without .NET Core 3.1